project/src/components/app/app.jsx

```diff
@@ -1,7 +1,16 @@
 import React from 'react';
+import MainPage from '../main-page/main-page';
+import PropTypes from 'prop-types';
 
-function App() {
-  return <p>Hello, world!</p>;
+function App(props) {
+  const {title, date} = props.data;
+  return (
+    <MainPage title = {title} date = {date}/>
```

Comment:
```suggestion
    <MainPage 
      title={title} 
      date={date}
    }/>
```

Не принято писать в одной строке, а также ставить пробелы у оператора присвоения. Также, тебе никто не мешает сделать так `<Main {...props} />`

Раз ты пропсы передаешь дальше.